packages/async-operations/tests/test-async_operations.js

```diff
@@ -61,7 +61,7 @@ test.before(async (t) => {
   t.context.functionConfig = {
     Environment: {
       Variables: {
-        ES_HOST: 'es-host',
+        Timeout: 300,
```

Comment:
What's this for?

Reply:
Since we're moving away from ES in general, ES indexing and all that, in the async operations tests I've changed all the places we're referencing ES operations to other options in those enums. Since we should probably be doing operations other than ES ones if ES is going away.

Reply:
So just as I changed the `operationType`s away from ES Index, I'm trying to move away from the `ES_HOST`. I wanted to keep a variable in here and found that Timeout could be one that is used.

Reply:
I see. And being that this is a mock, we just want to prove that we get back some semi-realistic config object.